ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -111,6 +135,24 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
+- as: aws-ipi-disc-priv-arm-mixarch-f28
```

Comment:
If I remember right, the e2e profile - `aws-ipi-disc-priv-arm-mixarch` is to cover mixarch nodes installation, the base profile `aws-ipi-disc-priv` is well covered by others in signle-arch ci. multi-arch payload image is very big compared with signle-arch, the upgrade ci will take more time/storage to run mirror images, I do not think we need to cover it. cc @shellyyang1989 for insight.

Reply:
Yes, we have both single-arch job on amd64 and arm64, so I agree to remove this one in multi

Reply:
Yeah, it's fine with me. @jianlinliu I'd like to confirm the rule further. Do you think which one is better?
a. not cover disc-mixarch in upgrade
b. only cover one disc-mixarch job in upgrade
c. any of your additional suggestions
I'm not sure if there is no any difference between single-arch and multiarch in image mirroring. If no difference, option a is fine with me. Otherwise, option b is fine.

Reply:
I prefer a.

>  if there is no any difference between single-arch and multiarch in image mirroring.

So far, I did not seem much difference, and at least, we already have disc-mixarch e2e ci jobs to cover the multi-arch payload mirror step, but I think we do not need to do it again in upgrade ci for these disc-mixarch install profile.

Reply:
ah, good point! So @JianLi-RH sounds like we can skip disc*mixarch profiles